texastribune/donations-app
# add autotrack, cross-domain, refactor click events

#### What's this PR do?

Add autotrack + cross-domain autoLink tracking
#### Why are we doing this? How does it help us?

GA goal conversion for FMD16 to track users across support.texastribune, checkout.texastribune and texastribune.org
#### How should this be manually tested?

Run the project following the readme docs, check that autotrack.js is loaded on the pages. Do a project search for onclick events, and make sure there aren't any `<a href>` with onclick tracking events. They should all be ga-event="click" with other data-labels per autotrack event tracking docs.
#### Have automated tests been added?

no
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

no
#### What are the relevant tickets?
#### How should this change be communicated to end users?
#### Next steps?
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note
